node_lambnik/src/database/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM quay.io/azavea/postgis:2.3-postgres9.6-slim
+
+RUN apt-get update
+RUN apt-get install postgis -y
```

Comment:
This line is a little curious since postgis should be available as built in the base image: https://github.com/azavea/docker-postgis/blob/master/Dockerfile-slim.template

Is it that the shp2pgsql tool isn't available via base?

Reply:
Exactly, I couldn't locate the package except as part of postgis.

Reply:
[According to this](https://gis.stackexchange.com/questions/148524/i-have-not-found-shp2pgsql-in-postgis-installation) shp2pgsql only comes with postgis-client, which isn't on the image. I'll use that instead of the entirety of postgis.

Reply:
Scratch that, [it looks like the supported way of installing shp2pgsql is to do exactly what I did](https://trac.osgeo.org/postgis/wiki/UsersWikiPostGIS23UbuntuPGSQL96Apt#Install).

Reply:
Ah, great then.  I was afraid we were installing potentially mixed versions of the database extension.